src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -616,6 +616,34 @@ namespace cryptonote
     return true;
   }
   //-----------------------------------------------------------------------------------------------
+  std::pair<uint64_t, uint64_t> core::get_coinbase_tx_sum(const uint64_t start_offset, const uint64_t count)
+  {
+    std::list<block> blocks;
+    std::list<transaction> txs;
+    std::list<crypto::hash> missed_txs;
+    uint64_t coinbase_amount = 0;
+    uint64_t emission_amount = 0;
+    uint64_t total_fee_amount = 0;
+    uint64_t tx_fee_amount = 0;
+    this->get_blocks(start_offset, count, blocks);
```

Comment:
Looking at the already_generated_coins function, does this return only the emission portion? I'm also moving the calculations to core_rpc_server in order to not pollute cryptonote_core, and looping over the blocks one at a time to get tx fees. 

You think I should just minimize the scope of the function to just show newly minted coins, which should be pretty quick?

Thanks again for your time/help.